packages/connectivity/connectivity/example/.gitignore

```diff
@@ -0,0 +1 @@
+generated_plugin_registrant.dart
```

Comment:
Reset all the changes to connectivity/connectivity. They need to be brought back post-federation.

Reply:
Done.